sdk/src/account.rs

```diff
@@ -3,6 +3,7 @@ use pubkey::Pubkey;
 /// An Account with userdata that is stored on chain
 #[repr(C)]
 #[derive(Serialize, Deserialize, Debug, Clone, Default)]
+#[serde(rename_all = "camelCase")]
```

Comment:
This is the only real change in this PR, rest are doc updates.

Slightly sketchy as we'll run into a conflict when another serialization format is desired elsewhere, but I don't see any better alternatives right now

Reply:
one alternative is to name `program_id` to `program` and `loader_id` to `loader`, but I don't have a strong opinion on that

Reply:
On second thought, how about renaming `program_id` to `owner`?

Reply:
Oh, good idea. I like the suggestion to drop the `_id`s, and do prefer `owner` to `program`